src/qinterface/gates.cpp

```diff
@@ -123,7 +123,7 @@ void QInterface::IPhaseRootN(bitLenInt n, bitLenInt qubit)
         return;
     }
 
-    ApplySinglePhase(ONE_CMPLX, pow(-ONE_CMPLX, -ONE_R1 / (ONE_BCI << (n - 1U))), true, qubit);
+    ApplySinglePhase(ONE_CMPLX, pow(-ONE_CMPLX, -ONE_R1 / (pow2(n - 1U))), true, qubit);
```

Comment:
So you want to replace these (and other) 1U with ONE_BCI as well?

Reply:
`n` is actually a `bitLenInt`, in this case, which is the only valid argument to `pow2()`, which returns a `bitCapInt`. These are therefore correct, in this case, but thank you for looking out!

Reply:
Point of `pow2()` is just to wrap a `bitLenInt` input safely to return its power of 2 as a `bitCapInt`, I'm saying.